resources/wikia/ui_components/modal/css/modal_default.scss

```diff
@@ -227,12 +224,13 @@ body.with-blackout {
 	}
 
 	.global-navigation {
-		position: static;
+		position: fixed;
 	}
 
 	.WikiaSiteWrapper {
 		position: fixed;
 		width: 100%;
+		z-index: 2;
```

Comment:
Not for this, see https://github.com/Wikia/app/pull/8227/files#diff-01b6b4c2965acf4b57d1e418b901c041R163